hq/app/services/CacheService.scala

```diff
@@ -112,7 +126,7 @@ class CacheService(config: Configuration, lifecycle: ApplicationLifecycle, envir
     setUpQuartzScheduleJob(Cache.AWSInspector, "45 20 1,7,13,19 * * ?")
   }
 
-  private def setUpQuartzScheduleJob(cache: Cache.EnumVal, cronString: String) = {
+  private def setUpQuartzScheduleJob(cache: Cache.EnumVal, cronString: String): Unit = {
```

Comment:
Is this required? I thought we were keeping the obvservables and just adding the client caches?